pkg/apis/deployment/environmentvariables_test.go

```
@@ -99,9 +100,11 @@ func Test_getEnvironmentVariablesForRadixOperator(t *testing.T) {
 			})
 		})
 		//goland:noinspection GoUnhandledErrorResult
-		testEnv.kubeUtil.CreateConfigMap(corev1.NamespaceDefault, &corev1.ConfigMap{ObjectMeta: metav1.ObjectMeta{Name: "radix-config"}, Data: map[string]string{
+		if _, err := testEnv.kubeUtil.CreateConfigMap(corev1.NamespaceDefault, &corev1.ConfigMap{ObjectMeta: metav1.ObjectMeta{Name: "radix-config"}, Data: map[string]string{
 			"clustername": testClusterName,
-		}})
+		}}); err != nil {
+			log.Error(err)
```

Use require.NoError to fail test instead of log

---

Fixed